### PR TITLE
8176033: New cygwin grep does not match \r as newline

### LINKED ISSUE #78: Refresh repo from upstream
There is a recent change for Windows that is blocking us from adding Windows JDK8 builds at https://ci.eclipse.org/openj9/
Please update the openj9 branch with the new new changes

Link to the change: http://hg.openjdk.java.net/jdk8u/jdk8u/rev/a05ed58d5ae0
```
author | kevinw
-- | --
Sat, 07 Apr 2018 03:28:39 -0700 (3 weeks ago)
cddffb220808
2209644bcac4
common/autoconf/basics_windows.m4 common/autoconf/generated-configure.sh
2 files changed, 5 insertions(+), 5 deletions(-) [+]

-    # The cmd output ends with Windows line endings (CR/LF), the grep command will strip that away
-    cygwin_winpath_root=`cd / ; cmd /c cd | grep ".*"`
+    # The cmd output ends with Windows line endings (CR/LF)
+    cygwin_winpath_root=`cd / ; cmd /c cd | $TR -d '\r\n'`
```

Blocking:
- eclipse/openj9#1662
- eclipse/openj9#1778

### PR DESCRIPTION
**Pull request by AdamBrousseau**
- Reviewed-by: erikj, tbell, ihse
- Original OpenJDK change: 2162:a05ed58d5ae0
- Applying this change before we pull it from upstream
- Change to translate delete command

Fixes #78

Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
I'm not entirely sure if I should be adding the copyright header since it doesn't appear that anyone previously changing the generated-configure.sh has added it.

File: common/autoconf/basics_windows.m4 [hunk 1]
```diff
@@ -23,6 +23,25 @@
 # questions.
 #
 
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018, 2018 All Rights Reserved
+# ===========================================================================
```
Comment (keithc-ca) [inline common/autoconf/basics_windows.m4]: Our convention is to insert just these three lines before the existing copyright; see common/autoconf/configure for an example.

**Comment by jdekonin** [comment]
Its not actually your change, so I don't think adding an IBM copyright is valid.

**Comment by AdamBrousseau** [comment]
Jenkins compile win,xlinux

**Comment by AdamBrousseau** [comment]
Jenkins compile win

**Comment by AdamBrousseau** [comment]
Jenkins compile win

**Comment by AdamBrousseau** [comment]
Jenkins compile win

**Comment by AdamBrousseau** [comment]
Jenkins compile win

**Comment by AdamBrousseau** [comment]
So the Windows compile actually gets past this piece of the code (configure). So this is new territory to debug on this new machine. We may end up needing more changes.

**Pull request merged**